packages/ocapn/src/client/index.js

```diff
@@ -0,0 +1,586 @@
+// @ts-check
+
+/**
+ * @typedef {import('../cryptography.js').OcapnPublicKey} OcapnPublicKey
+ * @typedef {import('../cryptography.js').OcapnKeyPair} OcapnKeyPair
+ * @typedef {import('../codecs/components.js').OcapnPublicKeyData} OcapnPublicKeyData
+ * @typedef {import('../codecs/components.js').OcapnLocation} OcapnLocation
+ * @typedef {import('../codecs/components.js').OcapnSignature} OcapnSignature
+ * @typedef {import('./types.js').Session} Session
+ * @typedef {import('./types.js').Connection} Connection
+ * @typedef {import('./types.js').Client} Client
+ * @typedef {import('./types.js').NetLayer} NetLayer
+ * @typedef {import('./types.js').SelfIdentity} SelfIdentity
+ * @typedef {import('./types.js').Logger} Logger
+ * @typedef {import('./types.js').LocationId} LocationId
+ * @typedef {import('./types.js').PendingSession} PendingSession
+ * @typedef {import('./types.js').SessionManager} SessionManager
+ * @typedef {import('./ocapn.js').GrantTracker} GrantTracker
+ * @typedef {import('./ocapn.js').Ocapn} Ocapn
+ */
```

Comment:
Can use `/** import {} from "" */` notation now, which is more transparent to the language server since it doesn’t create a new type.